transit_odp/timetables/loaders.py

```diff
@@ -222,3 +232,27 @@ def load_service_patterns(self, services, revision):
 
         adapter.info("Finished loading service patterns.")
         return service_patterns
+
+    def load_booking_arrangements(self, services, revision):
+        adapter = get_dataset_adapter_from_revision(logger, revision=revision)
+        booking_arrangements = self.transformed.booking_arrangements
+        if not booking_arrangements.empty:
+            booking_arrangements_reset = booking_arrangements.reset_index()
+            services_reset = services.reset_index()
+
+            merged_df = booking_arrangements_reset.merge(
+                services_reset[["service_code", "id"]], on=["service_code"], how="left"
```

Comment:
Could we merge using file_id so that if same service_codes are uploaded in different files then the appropriate service id is mapped to the booking arrangements